### PR TITLE
[DE] Add `People`, `People\peppy`, `People\The_Team\Account_support_team`; Update `People\The_Team\Support_Team` and `People\The_Team\Global_Moderation_Team`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
The member lists in `People\The_Team\Global_Moderation_Team` were left untranslated because it is just too much work and I don't see sense in translating stuff which might be outdated already next month.

I unfortunately needed to add a TODO to `People\The_Team\Account_support_team` and `People\The_Team\Global_Moderation_Team` because some articles which were linked from these articles aren't available yet.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Comment by The-Last-Cookie** [comment]
@respektive / @Phippe / @FamousPig do you want to review this?

File: wiki/People/The_Team/Account_support_team/de.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+- [Beschwerden für Einschränkungen auf deinem Account](/wiki/Help_Centre/Account_Restrictions) sowie zusätzlich:
+  - Stummschaltungen;
+  - Entfernung von Profilinhalten;
+  - Entfernung von Beatmaps;
+  - Entfernung von Forumsbeiträgen und Kommentaren.
+- Fälle von Fehlverhalten, die du [gestehen möchtest](/wiki/Reporting_Bad_Behaviour/Handling_Foul_Play#was-kann-ich-tun,-wenn-ich-die-Regeln-gebrochen-habe?).
+- [Verlust des Zugriffs auf die E-Mail](/wiki/Help_Centre/Installation_and_registration#wo-registriere-ich-mich-für-osu!?), die mit deinem osu!-Account verbunden ist oder, wenn dein Account gestohlen wurde.
+- [Rücksetzungen des Benutzernamens und Rechtschreibfehlerkorrekturen](/wiki/Help_Centre/Account#name-changes).
+- [Einschränkungen eines Bot-Accounts](/wiki/Bot_account).
```
Comment (respektive) [inline wiki/People/The_Team/Account_support_team/de.md:33]:
```suggestion
- [Registrierungen von Bot-Accounts](/wiki/Bot_account).
```

File: wiki/People/The_Team/Global_Moderation_Team/de.md [hunk 2]
```diff
@@ -1,142 +1,198 @@
...
+  - Moderatoren
 ---
 
 # Globales Moderationsteam
 
-**Globales Moderationsteam** (früher auch bekannt als Adminstratoren, Globales Modteam oder “GreenBAT”), allgemein auch **GMT**, sind Mitglieder des osu!Staffs, die in erster Linie für die Moderation der Channel und für Fragen, die auf das Personal und die gesamte Gemeinschaft gerichtet sind, zuständig sind.
+*Siehe auch: [Mod (Begriffsabgrenzung)](/wiki/Disambiguation/Mod).*
 
-## Ein GMT werden
+**Globales Moderationsteam** (früher auch bekannt als Globales Modteam, Administratoren oder “GreenBAT”), allgemein auch **GMT**, sind Mitglieder der osu!-Belegschaft, die in erster Linie für die Moderation der Spieler sowie einschlägige Probleme im Team und in der gesamten Community zuständig sind. Sie sind anhand ihres Nutzertitels und ihrer Nutzerfarbe, die rot im Spiel und Zitrusgrün in den Foren ist.
```
Comment (respektive) [inline wiki/People/The_Team/Global_Moderation_Team/de.md:14]:
```suggestion
**Globales Moderationsteam** (früher auch bekannt als Globales Modteam, Administratoren oder “GreenBAT”), allgemein auch **GMT**, sind Mitglieder der osu!-Belegschaft, die in erster Linie für die Moderation der Spieler sowie einschlägige Probleme im Team und in der gesamten Community zuständig sind. Sie sind anhand ihres Nutzertitels und ihrer Nutzerfarbe, die rot im Spiel und Zitrusgrün in den Foren ist, zu erkennen.
```

File: wiki/People/The_Team/Global_Moderation_Team/de.md [hunk 3]
```diff
@@ -1,142 +1,198 @@
...
+***Tipp: Um das Fehlverhalten eines Moderators zu melden, kontaktiere das [Account-Support-Team](/wiki/People/The_Team/Account_support_team#support@ppy.sh).***\
+*Siehe auch: [Melden von schlechtem Verhalten](/wiki/Reporting_Bad_Behaviour)*
+
+Das globale Moderationsteam ist für das Wohlergehen der Chats/Foren zuständig und kümmern sich um reibungslose Interaktionen innerhalb der Community. Du solltest dich an sie wenden, wenn du Fragen im Bezug auf die Moderationsaspekte des Spiels hast, das würde auch (aber nicht beschränkt auf) miteinbeziehen:
+
+- Probleme im Chat (z. B. Spam oder Belästigung)
+- Fehlverhalten in den Foren (z. B. unangemessene Beiträge)
+- die Beatmap-Diskussionen und Kommentarsektionen
+- Moderation der Benutzerprofile (NSFW oder anderweitig anstößige Inhalte in Avataren, Profilbilder oder -details, ...)
```
Comment (respektive) [inline wiki/People/The_Team/Global_Moderation_Team/de.md:28]:
```suggestion
- Moderation der Benutzerprofile (NSFW oder anderweitig anstößige Inhalte in Avataren, Profilbanner oder -details, ...)
```

**Review by respektive — CHANGES_REQUESTED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**